doc/nvim-treesitter.txt

```diff
@@ -449,228 +449,232 @@ Note: This is highly experimental, and folding can break on some types of
 ==============================================================================
 HIGHLIGHTS                                        *nvim-treesitter-highlights*
 
-							     *hl-TSAnnotation*
-`TSAnnotation`
-For C++/Dart attributes, annotations that can be attached to the code to
-denote some kind of meta information.
+The following is a list of highlights groups, the syntactic elements they
+apply to, and some examples.
 
-							      *hl-TSAttribute*
+							     *hl-TSAnnotation*
```

Comment:
I think the tag does not fit the header. Both should be `hl-TSAttribute`/`TSAttribute`

Reply:
Whoops. Missed that.